emails/email_alert/fr/html.ejs

```diff
@@ -1,11 +1,7 @@
-<p>Bonjour <%= user.name %>,</p>
+<p>Cher <%= user.name %>,</p>
+<p>Vous avez reçu ce message car un e-mail supplémentaire, <%= emailAdded %>, a été ajouté à votre profil à des fins de connexion.</p>
+<p>Si vous avez pris cette mesure, veuillez ne pas tenir compte de cet e-mail.</p>
+<p>Cependant, si vous n'avez pas ajouté cette adresse e-mail ou si l'adresse ci-dessus n'est pas votre adresse e-mail, veuillez vous connecter immédiatement à votre compte pour tout vérifier.</p>
+<p>Bien cordialement,</p>
 
-<p>Vous recevez ce message parcequ'une nouvelle adresse email, <%= emailAdded %>, a été ajoutée à votre profil Humanitarian ID.</p>
-
-<p>Si vous avez effectué cette action ou demandé à un manager Humanitarian ID de l'effectuer pour vous, vous pouvez ignorer ce message.</p>
-
-<p>En revanche, si l'adresse email citée ci-dessus ne vous appartient pas, contactez nous immédiatement à info@humanitarian.id !</p>
-
-<p>Cordialement,</p>
-
-<% include ../../includes/footer_fr %>
+<% include ../includes/footer_fr %>
```

Comment:
```suggestion
<% include ../../includes/footer_fr %>
```

This results in the alert emails not to be sent. **Note**: this is included in https://github.com/UN-OCHA/hid_api/pull/320

Reply:
> I couldn't find anywhere in the UI where to change an account's locale, so it looks like the French emails are only there for legacy accounts, is that correct?

That's correct. It's a vestigial HID Contact feature, but for now we're honoring the setting without offering a place to change it. All new accounts default to English.

Reply:
Nice catch here, btw. It would have been a fresh regression. Now it will never see the light of day 👍